hydra/test_utils/launcher_common_tests.py

```diff
@@ -346,6 +391,20 @@ def sweep_two_config_groups(
 
 @pytest.mark.usefixtures("hydra_restore_singletons")
 class IntegrationTestSuite:
+    def post_run_callback(self, tmp_dir: str) -> None:
```

Comment:
Since this is happening after the run, we should have the JobReturn. it seems much better to pass that than "tmp_dir".
This is what JobReturn contains:
```python
@dataclass
class JobReturn:
    overrides: Optional[Sequence[str]] = None
    return_value: Any = None
    cfg: Optional[DictConfig] = None
    hydra_cfg: Optional[DictConfig] = None
    working_dir: Optional[str] = None
    task_name: Optional[str] = None
```

Reply:
nice, yea, I wasn't sure what's best to put in the signature, this looks much better.

Reply:
Ok, I realized that I could just download the output from as part of running aws launcher so I've removed this completely.

This was not possible a few iterations ago because of reasons that I no longer recall 🙃 but refactoring helped make it possible.

Reply:
great.